ckanext/pages/theme/resources/styles/pages.css

```diff
@@ -15,3 +14,6 @@
 .page-list-item h3 a {
   display: block;
 }
+.page-simple-form-height {
+  height: 400px !important;
```

Comment:
I'm not a fan of `!important`. Wouldn't it be better to target the form via its ID (`#field-content`) in addition with a custom class?

Also, `page-simple-form-height` as a name doesn't tell me much. I'd also prefer a `pages` prefix for namespacing. How about `pages-text-editor`?

Reply:
It has nothing to do with the other editors. In fact there the pixel heights are set and for the plain text editor no height is set which is what I am trying to improve.

I think !important here is justified as we are overriding the normal CKAN behaviour as we are using this form in a weird way. It might have been/be better overall to not use this macro to create the form.

With page vs. pages - I am following the convention in the existing CSS. Simple-form-height seems a lot more precise to me vs. text-editor (see comment above).

Regarding the first commit, that is me pulling in the latest ckanext-pages changes, but in a rather stupid way via. the GitHub UI.